### PR TITLE
[node-red] migrate to common library

### PR DESCRIPTION
**Pull request by onedr0p**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/node-red/README.md [hunk 1]
```diff
@@ -1,107 +1,77 @@
-# Node-RED
+# Node-Red
 
-Low-code programming for event-driven applications
-
-**This chart is not maintained by the Node-RED project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new)**
```
Comment (bjw-s) [inline charts/node-red/README.md:5]: This was actually requested by Node-RED here: https://discourse.nodered.org/t/node-red-helm-chart/3784/3 (even though it's been 2 years now...)

File: charts/node-red/values.yaml [hunk 2]
```diff
@@ -1,132 +1,28 @@
...
-# Declare variables to be passed into your templates.
-
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
 
 image:
   repository: nodered/node-red
-  tag: 1.0.6-12-minimal
   pullPolicy: IfNotPresent
+  tag: 1.2.5
```
Comment (bjw-s) [inline charts/node-red/values.yaml:6]: I don't know what the reasoning used to be to go for the `minimal` variant. But it also makes sense to me to go for the 'default' image by default and if people want to use another variant they can override the value

Comment (onedr0p) [reply charts/node-red/values.yaml:6]: agreed!

**Review by bjw-s — APPROVED** [review]
LGTM 👍

**Pull request merged**